bcipy/helpers/save.py

```diff
@@ -169,3 +182,35 @@ def save_model(model: SignalModel, path: Union[Path, str]):
         # It supports very large objects and some data format optimizations
         # making it appropriate for signal models.
         pickle.dump(model, file, protocol=4)
+
+
+def save_stimuli_position_info(
+        stimuli_position_info: dict,
+        path: Union[Path, str],
+        screen_info: dict) -> str:
+    """Save stimuli positions and screen info to `path`
+
+   stimuli_position_info: {'A': (0, 0)}

```

Comment:
Same question here, I think it would be more informative if we mentioned the units here (pyschopy, Tobii or maybe just the screen limits like (0,0), (1,1), (-1,-1))

Reply:
Good idea, I'm happy to put those in the export as well. We will need to to some testing to see how these line up. I should have data to look at by EOD today.